docs/user_guide/02b_query_engine.md

```diff
@@ -149,42 +149,36 @@ lines: 41-56
 :::
 
 :::{note}
-Query methods used for changing the operands cannot be concatenated or assigned to variables, since their Return is None. That is, the following code snippet will set `gender_lowercase` as None, and as a result, an AttributeError will be thrown:
+Query methods used for changing the operands cannot be assigned to variables, for further querying, since their _Return_ is `None`. That is, the following code snippet will set `gender_lowercase` as None, and as a result, an `AttributeError` will be thrown:
```

Comment:
```suggestion
Query methods used for changing the operands cannot be assigned to variables, for further querying, since their return type is `None`. That is, the following code snippet will set `gender_lowercase` as None, and as a result, an `AttributeError` will be thrown:
```

Reply:
The `that is` sounds a bit weird at the beginning. You didn't change it but you're already here